### ISSUE TITLE
ARIA conventions

### ISSUE THREAD

**Issue by ain** [interactive-pioneers/conventions#90]
Basic [ARIA](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA) conventions that should be followed on every project in order to offer acceptable accessibility for people with disabilities.

**Comment by davleh** [comment]
There is already a small ARIA-part on the HTML conventions page. Is there a need to extend this section and to what extend? @GeorgMeyer23 @mpeters1
